package-builder/build-manifest.nix

```diff
@@ -162,7 +162,7 @@ pkgs.runCommandNoCC name
                 # N.B. not using t3 --forcecolor option because Nix sandbox
                 # strips color codes from output anyway.
                 FLOX_SRC_DIR=$(pwd) FLOX_RUNTIME_DIR="$TMP" \
-                  ${flox-env-package}/activate --env ${flox-env-package} --mode run --turbo -- \
+                  ${flox-env-package}/activate --env ${flox-env-package} --env-project "$PWD" --mode run --turbo -- \
```

Comment:
Is PWD correct even if the flox build command is run from another directory with `flox buid -d ...`?

Reply:
Yes, since the directory that the build is running in is `/build` and this is referencing that path, not the directory that the build was invoked from.